node/test/fixtures/mongodb/data-default.js

```diff
@@ -14,6 +14,11 @@ module.exports = {
       lessonId: 'lesson1',
       name: 'lesson name',
       image: 'some/image.png',
+      media: {
+        link: 'some/image.png',
+        type: 'image',
+        props: [],
```

Comment:
Inside the Lesson/Media schema...don't you have props defined as a dict? That would be OK if you're doing a transform `Prop[] <=> Record<string, string>` as things go in and out of the db, but it seems like something else is happening?